sdk/containerregistry/azure-containerregistry/azure/containerregistry/_container_repository_client.py

```diff
@@ -3,7 +3,7 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
 # ------------------------------------
-from typing import TYPE_CHECKING
+from typing import TYPE_CHECKING, Any, Dict
```

Comment:
nit: there doesn't seem to be a reason to move this out of the TYPE_CHECKING block, at least looking at the changes you made in this pr

Reply:
Correct, I learned that all imports from typing can be grouped together so I made that change here while it was fresh in my mind.